### ISSUE TITLE
Authentication is no longer working

### ISSUE THREAD

**Issue by Eric-Arellano** [Qiskit/gh-actions#51]
This step has failed for both qiskit/documentation and qiskit/qiskit_sphinx_theme at this step:

https://github.com/Qiskit/gh-actions/blob/28d8de27cc1aa2d8178539197a4dd32b9fe574f9/.github/workflows/code-engine-preview.yml#L124-L126

The failure looks like this:

```
ibmcloud login -g "Quantum Community" -r "us-south" -c "***" --apikey "***"
  shell: /usr/bin/bash -e {0}
  env:
    PREVIEW_TAG: pr-573
    APP_NAME: qiskit-sphinx-theme-pr-573
    DOCKER_IMAGE_TAG: icr.io/community-digital/qiskit-sphinx-theme:pr-573
API endpoint: https://cloud.ibm.com/
Authenticating...
OK

Targeted account SALVADOR DE LA PUENTE's Account (***) <-> 2317458

                   
API endpoint:      https://cloud.ibm.com/
FAILED
Region:            
Could not get resource group:
The following error(s) occurred:
The token does not have permission to perform this action.

Trace ID: d228a2a3-a794-4e3b-9610-e0f86e0814d2

User:              va@us.ibm.com
Account:           SALVADOR DE LA PUENTE's Account (***) <-> 2317458
Resource group:    No resource group targeted, use 'ibmcloud target -g RESOURCE_GROUP'
CF API endpoint:   
Org:               
Space:
```

The first failure was on October 27. 

The failing ibmcloud version was ibmcloud version 2.21.0+c3b7075-2023-10-24T15:37:53+00:00. The last working version was ibmcloud version 2.20.0+f382323-2023-09-19T20:06:39+00:00, so it's plausible that this failure is due to the new version of IBM Cloud CLI.

**Comment by Eric-Arellano** [comment]
This is due to https://github.com/IBM-Cloud/ibm-cloud-cli-release/issues/202.

**Comment by vabarbosa** [comment]
resolved by #52 #53 (downgraded to previous `ibmcloud` CLI)
